packages/bootstrap/scss/_variables.scss

```diff
@@ -2984,6 +2984,21 @@ $gantt-task-selected-text: $selected-text !default;
 $gantt-task-selected-border: $selected-bg !default;
 $gantt-task-progress-selected-bg: $selected-bg !default;
 
+$gantt-delayed-bg: $error !default;
+$gantt-delayed-bg-lighter: tint($gantt-delayed-bg, 5) !default;
+$gantt-advanced-bg: $success !default;
+$gantt-advanced-bg-lighter: tint($gantt-advanced-bg, 5) !default;
+
+$gantt-planned-color: $accent !default;
+$gantt-planned-dependencies-offset: .6em !default;
```

Comment:
use $gantt-planned-dependency-margin-x and $gantt-planned-dependency-margin-y variables for defining padding/margins